src/ray/common/cgroup2/tests/BUILD.bazel

```diff
@@ -31,3 +32,22 @@ ray_cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+# This test is run through sysfs_cgroup_driver_integration_test_entrypoint.sh
```

Comment:
leave instructions for how to run them manually (if they're in the shell script, just mention it here)